index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Glory Mugizi Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="navbar">
+          <div class="my-logo"><a href="#">My Logo</a></div>
+          <div class="toggle-menu">
+            <img src="images/humbger.png" alt="toggle menu" />
+          </div>
+        </div>
+      </nav></header>
+      <section class="headline">
+        <div class="container heading">
+          <h1>I’m Glory Mugizi Glad to see you!</h1>
```

Comment:
- [ ] As the previous code reviewer was explaining , **always make sure to stick as much as possible to the Figma design. You should use the Figma design as a reference for your ```colors``` , ```font-size```, ```margins```, ```paddings```, ```font-weight```, and so forth** :100:. You did a great job using the same ```colour``` and ```font-size``` :+1:  however, the ```font-weight``` you use here should be the same as in the Figma design. 

Your design : 

![Screenshot from 2022-12-02 10-46-44](https://user-images.githubusercontent.com/80458199/205257321-96e06b2e-4295-4bdf-a77c-da01d2ebf69b.png)

The Figma design:

![Screenshot from 2022-12-02 10-53-55](https://user-images.githubusercontent.com/80458199/205257416-6c415bf4-d2df-4efb-ac5b-3274447eb6e0.png)

Reply:
@glorymugizi, Kindly stick to the Figma design

Reply:
Make sure the font-weight is the same

Reply:
Hi @iambenkis,
Changed the logo font size to 18px  and the heading as per your requested suggestions.

Thank you.

Reply:
Fixed.